README.md

```diff
@@ -46,21 +48,26 @@ Then follow the instructions in [INSTALL](INSTALL.md).
 
 ## Configuration
 
-For necessary server-side configurations, follow instructions in
-[INSTALL - Configuration](INSTALL.md#configuration).
+If not using the vagrant box, follow instructions in
+[INSTALL - Stand alone installation](INSTALL.md#stand-alone-installation)
+for necessary server-side configurations.
 
 ## Usage
 
-Generally you will have four tabs (or windows) open in your terminal when using this project.
-These will be used for:
+When running this project you will generally have four tabs
+(or windows) open in your terminal, which will be used for:
  1. **Git operations**.
-    Perform Git operations and general development in the repository.
+    Perform Git operations and general development in the repository,
+    such as `git checkout flapjack`.
  2. **Elasticsearch**.
```

Comment:
Added general note at the top, like usage has https://github.com/cfpb/cfgov-refresh/pull/2098/commits/e01ffe3f7d648fd4a6bef42f171b1312afdb4109